src/components/SwapForm/SwapActionButton/index.tsx

```diff
@@ -0,0 +1,226 @@
+import { Currency, CurrencyAmount } from '@kyberswap/ks-sdk-core'
+import { Trans } from '@lingui/macro'
+import { useEffect, useState } from 'react'
+import styled from 'styled-components'
+
+import { ButtonConfirmed, ButtonLight, ButtonPrimary } from 'components/Button'
+import Column from 'components/Column/index'
+import Loader from 'components/Loader'
+import ProgressSteps from 'components/ProgressSteps'
+import { AutoRow, RowBetween } from 'components/Row'
+import SwapOnlyButton from 'components/SwapForm/SwapActionButton/SwapOnlyButton'
+import { BuildRouteResult } from 'components/SwapForm/hooks/useBuildRoute'
+import { SwapCallbackError } from 'components/swapv2/styleds'
+import { useActiveWeb3React } from 'hooks'
+import { ApprovalState, useApproveCallback } from 'hooks/useApproveCallback'
+import { WrapType } from 'hooks/useWrapCallback'
+import { useWalletModalToggle } from 'state/application/hooks'
+import { DetailedRouteSummary } from 'types/route'
+
+import { Props as SwapOnlyButtonProps } from './SwapOnlyButton'
+
+const CustomPrimaryButton = styled(ButtonPrimary).attrs({
```

Comment:
```suggestion
const SwapButton = styled(ButtonPrimary).attrs({
```

Reply:
Hm. It shouldn't be named SwapButton. We use that button in several cases such as Wrap, Unwrap, input error, Insufficient liquidity

Reply:
We already have SwapActionButton for general purposes (swapping/error/approve flow) and SwapOnlyButton for swapping only. 
SwapButton will make it more confusing I'm afraid